README.md

```diff
@@ -31,7 +40,8 @@ client affinity and can auto-reconnect to restarted nodes.
 Prerequisites
 =============
 
-- Boost C++ Library 1.53.0 or above.
+Boost C++ Library 1.53 if using the downloaded client library. If building from
+source, you may use 1.53 or above.
```

Comment:
I believe someone is going to try to build the binary, but link with a compiler different from the Centos 6 default GCC compiler.  If their compiler is older than the C6 compiler things might link.  If their's is older it probably won't link.

The issue is really the version of the standard C++ library.  This is not exactly coordinated with versions of GCC, but it's very closely related.